templates/repo/diff/comment_form.tmpl

```diff
@@ -22,7 +22,7 @@
 			</div>
 		</div>
 		<div class="field footer mx-3">
-			<span class="markup-info">{{svg "octicon-markup"}} {{$.root.i18n.Tr "repo.diff.comment.markup_info"}}</span>
```

Comment:
Or maybe we should rename the key?

Reply:
I'm against renaming key, because then it removes that from the translated strings and the translators need to all add that in again.